Composer/packages/client/src/components/ProjectTree/index.tsx

```diff
@@ -181,8 +184,13 @@ export const ProjectTree: React.FC<IProjectTreeProps> = props => {
           onChange={onFilter}
           iconProps={{ iconName: 'Filter' }}
         />
+        <div
+          aria-live={'polite'}
+          aria-label={`${res.groups.length} dialogs have been found
+          ${res.groups.length > 0 ? 'Press down arrow key to navigate the search results' : ''}`}
```

Comment:
Please use formatMessage so we can get translations of this. It would also be useful to take advantage of its built-in pluralization mechanism.

Reply:
done